sync-core/src/test/java/com/cloudant/sync/datastore/BasicDatastoreCRUDTest.java

```diff
@@ -140,21 +140,21 @@ private Map createMapWithSpecialField() {
     }
 
     @Test
-    public void createLocalDocument_bodyOnly_success() {
+    public void createLocalDocument_bodyOnly_success() throws Exception {
         BasicDocumentRevision rev = datastore.createLocalDocument(bodyOne);
         validateNewlyCreateLocalDocument(rev);
     }
 
     @Test
-    public void createLocalDocument_docIdAndBody_success() {
+    public void createLocalDocument_docIdAndBody_success() throws Exception {
         String docId = CouchUtils.generateDocumentId();
         BasicDocumentRevision rev = datastore.createLocalDocument(docId, bodyOne);
         validateNewlyCreateLocalDocument(rev);
         Assert.assertEquals(docId, rev.getId());
     }
 
-    @Test(expected = IllegalArgumentException.class)
-    public void createLocalDocument_existDocId_exception() {
+    @Test(expected = DocumentException.class)
```

Comment:
Can any of these be more specific. I think they should check for the sub-class they expect to get if they can be more specific.